crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -872,13 +870,7 @@ where
             // calls `_extends` or `Object.assign`
             Box::new(Expr::Call(CallExpr {
                 span: DUMMY_SP,
-                callee: {
-                    if self.use_builtins {
-                        member_expr!(DUMMY_SP, Object.assign).as_callee()
-                    } else {
-                        helper!(extends, "extends")
-                    }
-                },
+                callee: helper!(extends, "extends"),
```

Comment:
I think this should be reverted

Reply:
I don't quite understand the difference between configuring use_buildin and use_spread. It looks like the public is the same, only one is {... } One is Object.assign

Reply:
Maybe we can support issue without fixing it, just configure use_spread